src/ert/config/parsing/queue_system.py

```diff
@@ -9,19 +9,22 @@ class QueueDriverEnum(BaseCEnum):  # type: ignore
     LOCAL = None
     TORQUE = None
     SLURM = None
+    GENERAL = None
 
 
 QueueDriverEnum.addEnum("LSF", 1)
 QueueDriverEnum.addEnum("LOCAL", 2)
 QueueDriverEnum.addEnum("TORQUE", 4)
 QueueDriverEnum.addEnum("SLURM", 5)
+QueueDriverEnum.addEnum("GENERIC", 6)
 
 
 class QueueSystem(StrEnum):
     LSF = "LSF"
     LOCAL = "LOCAL"
     TORQUE = "TORQUE"
     SLURM = "SLURM"
+    GENERIC = "*"
```

Comment:
I saw the issue had a proposal of using `*` as wildcard, but an option could be to specify `GENERIC` instead.
I'm personally not especially fond of wildcards.

Reply:
I think ERT users are used to `*` as wildcards but I have no strong feelings towards wildcards so I can change it to `GENERIC = "GENERIC"`.